README.md

```diff
@@ -8,3 +8,23 @@ MapStore Project for geOrchestra
 See https://georchestra.geo-solutions.it/ (work in progress)
 
 
+Building the project
+------------
+
+Clone the repository:
+
+`git clone https://github.com/georchestra/mapstore2-georchestra.git`
```

Comment:
I think you have to add the option` --resursive` or suggest submodule init and update

Reply:
You are right. Fixed